bot/cogs/help.py

```diff
@@ -146,7 +147,13 @@ async def command_not_found(self, string: str) -> "HelpQueryNotFound":
         Will return an instance of the `HelpQueryNotFound` exception with the error message and possible matches.
         """
         choices = await self.get_all_help_choices()
-        result = process.extractBests(string, choices, scorer=fuzz.ratio, score_cutoff=60)
+
+        # Run fuzzywuzzy's processor beforehand, and avoid matching if processed string is empty
+        # This avoids fuzzywuzzy from raising a warning on inputs with only non-alphanumeric characters
+        if full_process(string):
```

Comment:
This returns a new string. You should pass it to `extractBests`. Good opportunity for a walrus here.

Reply:
Good catch.